cockpit/src/subscriptions-view.jsx

```diff
@@ -99,18 +99,29 @@ class InstalledProducts extends React.Component {
         let entries = this.props.products.map(function (itm) {
             let subscribed;
             let status_text;
+            let label_status_text;
             let start_date_text;
             let end_date_text;
             let body;
             let columns;
 
             if (itm.status === 'subscribed') {
-                subscribed = true;
-                status_text = _("Subscribed");
-
+                subscribed = "green";
+                label_status_text = _("Subscribed");
+                status_text = label_status_text;
+            } else if (itm.status === 'partially_subscribed') {
+                subscribed = "orange";
+                label_status_text = _("Partially subscribed");
+                status_text = label_status_text + ' (' + itm.status_details.join(',') + ')';
+            } else if (itm.status === 'not_subscribed') {
+                subscribed = "red";
+                label_status_text = _("Not subscribed");
+                status_text = label_status_text + ' (' + itm.status_details.join(',') + ')';
             } else {
-                subscribed = false;
-                status_text = _("Not subscribed (Not supported by a valid subscription)");
+                console.debug('Other state:', itm.status);
+                subscribed = "red";
+                label_status_text = _("Not subscribed");
```

Comment:
considering this case is no more for unsubscribed, should it rather say "Unknown status" or (something like that) to avoid confusion and to notice this situation better?